scripts/vm/hypervisor/xenserver/setup_heartbeat_sr.sh

```diff
@@ -72,7 +72,7 @@ else
       fi
       rm $link -f
     fi 
-    lvcreate VG_XenStorage-$2 -n hb-$1 --size 4M
+    lvcreate VG_XenStorage-$2 -n hb-$1 --size 4M  --config global{metadata_read_only=0}
```

Comment:
On 6.5 metadata_read_only is already set to false, so this just overrides with the same value.